roles/openshift_facts/library/openshift_facts.py

```diff
@@ -715,6 +715,26 @@ def set_version_facts_if_unset(facts):
 
     return facts
 
+def set_manageiq_facts_if_unset(facts):
+    """ Set manageiq facts. This currently includes common.use_manageiq.
+
+        Args:
+            facts (dict): existing facts
+        Returns:
+            dict: the facts dict updated with version facts.
+        Raises:
+            OpenShiftFactsInternalError:
+    """
+    if 'common' not in facts:
+        if 'version_greater_than_3_1_or_1_1' not in facts['common']:
+            raise OpenShiftFactsInternalError(
+                "Invalid invocation: The required facts are not set"
+            )
+
+    facts['common']['use_manageiq'] = facts['common']['version_greater_than_3_1_or_1_1']
```

Comment:
@detiber @brenton about `version_greater_than_3_1_or_1_1` (probably just about the naming), shouldn't be "equal or greater than"? (e.g. what about 3.1.0)